gemma-teixido/challenge/tour-of-heroes-fullstack/backend/src/helpers/index.js

```diff
@@ -0,0 +1,15 @@
+function helpers() {
```

Comment:
there is no need of these any more. Because you will get the id generated in mongodb